Support for Oruga's patch versions

Hi 👋🏻 I’m not sure if this is intentional, but it seems the Oruga peer dependency cannot currently be resolved due to an upstream dependency conflict. The theme-bulma requires version 0.9.0, while the latest Oruga version is 0.9.1 (since yesterday 😉).

<details><summary>npm ERESOLVE</summary>
<p>

```text
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: musema-spotlight@undefined
npm ERR! Found: @oruga-ui/oruga-next@0.9.1
npm ERR! node_modules/@oruga-ui/oruga-next
npm ERR!   @oruga-ui/oruga-next@"0.9.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @oruga-ui/oruga-next@"0.9.0" from @oruga-ui/theme-bulma@0.5.1
npm ERR! node_modules/@oruga-ui/theme-bulma
npm ERR!   @oruga-ui/theme-bulma@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

</p>
</details> 

While I understand the importance of locking major and minor versions, it might be worth considering whether all patch versions should be supported 🤔 

```json
"peerDependencies": {
  "oruga": "~0.9.0"
}
```

--- a/package.json
+++ b/package.json
@@ -31,14 +31,14 @@
     "update": "ncu -u"
   },
   "peerDependencies": {
-    "@oruga-ui/oruga-next": "0.9.0"
+    "@oruga-ui/oruga-next": "^0.9.0"
   },
   "dependencies": {
     "bulma": "1.0.2"
   },
   "devDependencies": {
-    "@oruga-ui/examples": "0.9.0",
-    "@oruga-ui/oruga-next": "0.9.0",
+    "@oruga-ui/examples": "0.9.1",
+    "@oruga-ui/oruga-next": "0.9.1",
     "@types/node": "^22.9.0",
     "@vitejs/plugin-vue": "5.1.4",
     "@vue/eslint-config-prettier": "^10.1.0",

